### ISSUE TITLE
How can I find the training code for the boby model?

### ISSUE THREAD

**Issue by aishangmgu** [CMU-Perceptual-Computing-Lab/openpose#1361]
How can I find the training code for the boby model?

**Comment by soulslicer** [comment]
I have a pytorch version that should match the training accuracy here:

https://github.com/soulslicer/pytorch_openpose

**Comment by gineshidalgo99** [comment]
https://github.com/CMU-Perceptual-Computing-Lab/openpose_train
